api/user.go

```diff
@@ -67,6 +65,11 @@ func InitUser(r *mux.Router) {
 	sr.Handle("/{id:[A-Za-z0-9]+}/sessions", ApiUserRequired(getSessions)).Methods("GET")
 	sr.Handle("/{id:[A-Za-z0-9]+}/audits", ApiUserRequired(getAudits)).Methods("GET")
 	sr.Handle("/{id:[A-Za-z0-9]+}/image", ApiUserRequired(getProfileImage)).Methods("GET")
+
```

Comment:
Hmmm, but these APIs are actions on the user object themselves. The user API file is getting a bit crowded though, so this is probably worth a discussion. Propose no action here, but discuss during the next dev meeting?